tests/unit/test_analytics.py

```diff
@@ -0,0 +1,9 @@
+from __future__ import annotations
+
+from ragas._analytics import add_userid
+
+
+def test_add_userid():
+    payload = {"a": 1, "b": 2}
+    payload = add_userid(payload)
```

Comment:
create a new instance of `BaseEvent` and check if it has the same user_id as from appdirs